package.json

```diff
@@ -25,7 +25,7 @@
     "pretty-quick": "pretty-quick --staged",
     "publish": "node utils/pre-publish-check.js && git diff --stat --exit-code HEAD && yarn build:cjs && yarn build:es6 && lerna publish",
     "start:prod": "yarn run build:styleguidist && yarn run styleguidist --config styleguide.prod.config.js",
-    "start:storybook": "start-storybook",
+    "start-storybook": "start-storybook -s ./static",
```

Comment:
I ended up changing this command to match `build-storybook`.  I was getting confused when I was running `yarn start-storybook` before this change since it was running `start-storybook` with out the `-s` option.